R/create_episode_file.R

```diff
@@ -73,9 +75,13 @@ create_episode_file <- function(
         "diag6",
         "op1a",
         "age",
+        "ch_name",
+        "ch_postcode",
+        "cup_pathway",
         "cij_marker",
         "cij_start_date",
         "cij_end_date",
+        "cij_pattype",
         "cij_pattype_code",
         "cij_ipdc",
         "cij_admtype",
```

Comment:
Im getting this error here: 

ℹ The file temp_ep_file_variable_store_2021.parquet did not exist in
/conf/sourcedev/Source_Linkage_File_Updates/2021, it has now been created.
Error in `check_variables_exist()` at source-linkage-files/R/create_episode_file.R:290:3:
! Variable "hscp" is required, but is missing from `data`.
Run `rlang::last_trace()` to see where the error occurred.